components/log-viewer-webui/server/src/DbManager.js

```diff
@@ -101,20 +110,43 @@ class DbManager {
     }
 
     /**
-     * Submits an IR extraction job to the scheduler and waits for it to finish.
+     * Submits a stream extraction job to the scheduler and waits for it to finish.
      *
-     * @param {object} jobConfig
+     * @param {number} jobType
+     * @param {number} logEventIdx
+     * @param {string} streamId
+     * @param {number} targetUncompressedSize
      * @return {Promise<number|null>} The ID of the job or null if an error occurred.
      */
-    async submitAndWaitForExtractIrJob (jobConfig) {
+    async submitAndWaitForExtractStreamJob (
+        jobType,
+        logEventIdx,
+        streamId,
+        targetUncompressedSize
+    ) {
```

Comment:
When there are multiple parameters to a single function, it's be better to use the object destructuring syntax:
```suggestion
    async submitAndWaitForExtractStreamJob ({
        jobType,
        logEventIdx,
        streamId,
        targetUncompressedSize
    }) {
```
The JsDoc can be updated as:
```
     * @param {object} props
     * @param {number} props.jobType
     * @param {number} props.logEventIdx
     * @param {string} props.streamId
     * @param {number} props.targetUncompressedSize
```

Then the method call can be modified as:
```
            const extractResult = await fastify.dbManager.submitAndWaitForExtractStreamJob({
                jobType: extractJobType,
                logEventIdx: sanitizedLogEventIdx,
                streamId: streamId,
                targetUncompressedSize: settings.StreamTargetUncompressedSize,
            });
```
which looks more readable. What do you think?